README.md

```diff
@@ -5,22 +5,31 @@
 
 # SDALib
 
-A framework to create AI for assisted cars for Linux and Windows. 
+A framework to create AI for assisted cars for Linux (64 bit) and Windows (32 bit) in C++ and Python.
 This Library can connect to the [DAISI](https://github.com/red-panda-productions/speed-dreams) simulation to adjust the controls of the human player in the simulation. The AI (currently) has full access to edit steer, acceleration and brake of the car.
 
 ## Download
 
 You can download the library from the [Releases](https://github.com/red-panda-productions/SDALib/releases) page on the repository. There you will find 2 different kind of packages:
 - premade (can be seen by having 'premade' in their file name)
+- premade-python
 - standalone
 
 Premade files contain premade projects so that you can immediately start with creating your AI. The standalone files are there for when you have an existing project that you want to add this library to.
 
 Furthermore, you can always create your project within the source code of SDALib by downloading or cloning the repository.
 
-## Using SDALib in your project
+## Using SDALib in your C++ project
 
-When you use the project, we suggest you use CMake and copy over the different .cmake files into a CMake folder in your repository. The SDAMacros.cmake file consists of macros like FIND_SDA_PACKAGES and INSTALL_SDA_PACKAGES(target) that can easily find and install all of the dependencies needed for SDALib into your project. After calling these macros the setup of SDALib is done and you can begin programming your AI.
+When you use the project on Windows you will need to use a 32 bit compiler, on Linux you can use 64 bit. Furthermore we suggest that you use CMake and that you copy over the different .cmake files into a CMake folder in your repository. The SDAMacros.cmake file consists of macros like FIND_SDA_PACKAGES and INSTALL_SDA_PACKAGES(target) that can easily find and install all of the dependencies needed for SDALib into your project. After calling these macros the setup of SDALib is done and you can begin programming your AI.
+
+## Using SDALib in your Python project
+
+On Windows you will need to have a 32 bit version of python installed, on Linux you can use 64 bit. When you use the python project you will get an executable and 2 files, SDATypes.py and SDADriver.py. You can build your AI in SDADriver.py. When you want to use your driver with [DAISI](https://github.com/red-panda-productions/speed-dreams) be sure to select the SDALib-Python.exe executable. The executable will load your driver into the Python runtime and will give data to your python driver
```

Comment:
```suggestion
On Windows you will need to have a 32 bit version of python installed, on Linux you can use 64 bit. When you use the python project you will get an executable and 2 files, SDATypes.py and SDADriver.py. You can build your AI in SDADriver.py. When you want to use your driver with [DAISI](https://github.com/red-panda-productions/DAISI) be sure to select the SDALib-Python.exe executable. The executable will load your driver into the Python runtime and give data to your python driver
```
- "When you use the python project" what python project?
- "SDATypes.py and SDADriver.py. You can build your AI in SDADriver.py." what about SDATypes?
- "will give data to your python driver" what data?